Zend/zend_alloc.c

```diff
@@ -1171,10 +1171,10 @@ static zend_always_inline int zend_mm_small_size_to_bit(int size)
 # define MIN(a, b) (((a) < (b)) ? (a) : (b))
 #endif
 
-static zend_always_inline int zend_mm_small_size_to_bin(size_t size)
+static zend_always_inline size_t zend_mm_small_size_to_bin(size_t size)
 {
-#if 0
-	int n;
+#if 0 // can we have a discussion about this `#if 0`?
```

Comment:
you should drop this comment and start the discussion with a comment in this PR.

Reply:
By "we", in this case, I think you know where I stand, so I'm going to sit out of the conversation. If you're going to talk about things you should drop, to be clear, **it's the entire block of code that's being unconditionally excluded there that should be dropped**, not just the comment...

Nonetheless, I'm no longer interested in PHP. I wasn't interested in the first place. I fully regret all of my contributions. They're likely to taint my name. I'm tempted to just delete it all, based on the way this is all dragging me back constantly. I want to work on things that aren't... unportable...